fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java

```diff
@@ -1163,7 +1195,11 @@ public TStreamLoadPutResult streamLoadPut(TStreamLoadPutRequest request) {
         TStatus status = new TStatus(TStatusCode.OK);
         result.setStatus(status);
         try {
-            result.setParams(streamLoadPutImpl(request));
+            if (request.getVersion() == 1) {
```

Comment:
the version variable may not be set for non-sql-load. Should check if it is set.